chrome/popup.js

```
@@ -0,0 +1,279 @@
+  let filteredBanList = [];
+  let currentTab = null;
+  let currentPage = 1;
+  const itemsPerPage = 10;
+  let searchTerm = '';
+  
+  // Get current tab
+  const tabs = await chrome.tabs.query({ active: true, currentWindow: true });
+  currentTab = tabs[0];
+  
+  // Check if we're on YouTube
+  const isYouTube = currentTab && currentTab.url && currentTab.url.includes('youtube.com');
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)